src/cli/ansi_terminal/ansi_term-basic.md

```diff
@@ -0,0 +1,39 @@
+## ANSI Terminal
+
+[![ansi_term-badge]][ansi_term] [![cat-command-line-badge]][cat-command-line]
+
+This program depicts the use of [`ansi_term` crate] and how it is used for controlling colours and formatting, such as blue bold text or yellow underlined text, on ANSI terminals.
+
+There are two main data structures in this crate that you need to be concerned with: ANSIString and Style. A Style holds stylistic information: colours, whether the text should be bold, or blinking, or whatever. There are also Colour variants that represent simple foreground colour styles. An ANSIString is a string paired with a Style.
```

Comment:
I thought I commented on this line yesterday, but I apologize for not getting you all the information at once.  We need to remove the personal pronouns.  Instead of talking about the author or the audience, please refer to the program.  Therefore, the you in this line needs to be removed.  Also, ANSIString and Style should be links styled as code.  For example: https://docs.rs/ansi_term/*/ansi_term/type.ANSIString.html